packages/common/src/anchor-service.ts

```diff
@@ -95,6 +95,11 @@ export interface AnchorService {
  * Describes behavior for validation anchor commit inclusion on chain
  */
 export interface AnchorValidator {
+  /**
+   * The rpc endpoint used to validate anchor transactions.
+   */
+  validationRpcEndpoint: string | null
```

Comment:
not necessarily, it might just be using the default provider from the ethers.js library (which is rate limited) instead of a dedicated ethereum rpc endpoint.

Reply:
Thanks! Could you add it this short explanation to the comment in line 99?

Reply:
Ah, you've already done it :). Sorry for not checking and thanks!